src/components/m-virtual-displays.vue

```diff
@@ -276,7 +298,22 @@ export default {
     this.displayType = this.defaultDisplays[0];
   },
   methods: {
+    isDisabled(item) {
+      return this.disabledItems.includes(item[this.keyField] )
+    },
+    isSelected(row) {
+      return this.selectedItems.some(item => item[this.keyField] === row[this.keyField] )
+    },
+    onItemSelectionChange(selectedItem) {
+      if (this.isSelected(selectedItem)) {
+        const index = this.selectedItems.findIndex(item => item[this.keyField] === selectedItem[this.keyField])
+        this.selectedItems.splice(index, 1)
+      } else {
+        this.selectedItems.push(selectedItem)
+      }
 
+      this.$emit('on-change-selection-row', { selectedItem, selectedItems: this.selectedItems })
```

Comment:
Comme on a changé la méthode faudrait mieux changer le emit aussi nan ?

```suggestion
      this.$emit('on-item-selection-change', { selectedItem, selectedItems: this.selectedItems })
```